xml/installation-installation-ironic-ironic_provisioning.xml

```diff
@@ -113,16 +114,17 @@
   <para>
    The <literal>ir-deploy-ramdisk-HOS4.0</literal> image is a traditional boot
    ramdisk used by the <literal>agent_ipmitool</literal>,
-   <literal>pxe_ipmitool</literal>, and <literal>pxe_ilo</literal> drivers while
-   <literal>ir-deploy-iso-HOS4.0</literal> is an ISO image that is supplied as
-   virtual media to the host when using the <literal>agent_ilo</literal> driver.
+   <literal>pxe_ipmitool</literal>, and <literal>pxe_ilo</literal> drivers
+   while <literal>ir-deploy-iso-HOS4.0</literal> is an ISO image that is
```

Comment:
Also ... just because of these formatting changes, I notice things I wouldn't have otherwise. E.g. this file name is probably wrong. :)